game.py

```diff
@@ -0,0 +1,32 @@
+import random
+while(True):
+    n=random.randint(1,100)
+    i=1
+    while(True):

```

Comment:
توی هر زبانی یه چیزی داریم تحت عنوان style guide و توی پایتون میتونی یه نگاه به PEP8 بندازی

توی استایل گاید ها برای خوانایی میگن که چه قوانینی رعایت بشن

مثلا این رو بهتره بنویسی 

```suggestion
    while True:
```